pkg/task/task.go

```diff
@@ -404,6 +407,8 @@ func (m *taskExecutor) ExecuteIt() (err error) {
 		err = m.patchOEV1alpha1CSV()
 	} else if m.metaTaskExec.isPatchOEV1alpha1CVR() {
 		err = m.patchOEV1alpha1CVR()
+	} else if m.metaTaskExec.isPatchOEV1alpha1UR() {
+		err = m.patchOEV1alpha1UR()
```

Comment:
Let us have this as `err = m.patchUpgradeResult()`

Reply:
Done